lib/vector-common/src/shutdown.rs

```diff
@@ -279,7 +279,7 @@ impl SourceShutdownCoordinator {
         )
     }
 
-    /// Returned future will finish once all *current* sources have finished.
+    /// Returned future will finish once all sources have finished.
```

Comment:
This note might be worth keeping, since it was a little confusing to find.

Reply:
Fixed in a61aa0cc1